ibu-imager/hack/clean.sh

```diff
@@ -1,13 +1,27 @@
 #!/bin/bash
 
-sudo rm -rf /var/tmp/container_list.done \
-    /var/tmp/backup  && \
-sudo rm -f /usr/local/bin/prepare-installation-configuration.sh \
-    /usr/local/bin/installation-configuration.sh && \
-sudo systemctl disable installation-configuration.service && \
-sudo systemctl disable prepare-installation-configuration.service && \
-rm -f /etc/systemd/system/installation-configuration.service \
-    /etc/systemd/system/prepare-installation-configuration.service && \
-sudo podman rmi quay.io/alosadag/ibu-seed-sno0:oneimage --force && \
-sudo systemctl enable --now kubelet && \
-    sudo systemctl enable --now crio
+# 1) cleanup backup dir
+sudo rm -rf /var/tmp/container_list.done /var/tmp/backup
+
+# 2) cleanup preparation scripts and services
+sudo rm -f /usr/local/bin/prepare-installation-configuration.sh /usr/local/bin/installation-configuration.sh
+sudo systemctl disable installation-configuration.service
+sudo systemctl disable prepare-installation-configuration.service
+rm -f /etc/systemd/system/installation-configuration.service /etc/systemd/system/prepare-installation-configuration.service
+
+# cleanup container images
+sudo podman rmi quay.io/alosadag/ibu-seed-sno0:oneimage --force
+
+# restore etcd
+RECERT_IMAGE="quay.io/edge-infrastructure/recert:latest"
+ETCD_IMAGE=$(jq -r '.spec.containers[] | select(.name == "etcd") | .image' </etc/kubernetes/manifests/etcd-pod.yaml)
+
+podman run --name recert_etcd --detach --rm --network=host --privileged --replace --authfile /var/lib/kubelet/config.json --entrypoint etcd -v /var/lib/etcd:/store ${ETCD_IMAGE} --name editor --data-dir /store
+podman run --name recert --rm --network=host --privileged --replace --authfile /var/lib/kubelet/config.json -v /etc/kubernetes:/kubernetes -v /var/lib/kubelet:/kubelet -v /etc/machine-config-daemon:/machine-config-daemon ${RECERT_IMAGE} --etcd-endpoint localhost:2379 --static-dir /kubernetes --static-dir /kubelet --static-dir /machine-config-daemon --extend-expiration
+
+podman rm -f recert_etcd
+podman rm -f recert
+
+# restore cluster services
+sudo systemctl enable --now kubelet
+sudo systemctl enable --now crio
```

Comment:
crio.service isn't actually enabled, so we don't need this last command. It gets started automatically by kubelet.service, I believe